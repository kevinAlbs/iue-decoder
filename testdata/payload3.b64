A1gAAAAQdAACAAAAEGEAAgAAAAVraQAQAAAABBI0VngSNJh2EjQSNFZ4kBIFa3UAEAAAAAQSNFZ4EjSYdhI0EjRWeJASEHYAQOIBABJjbQAAAAAAAAAAAAA=