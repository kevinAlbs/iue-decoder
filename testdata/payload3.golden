BlobSubtype=3
FLE2EncryptionPlaceholder_Type=Find ("t":2)
FLE2EncryptionPlaceholder_Algorithm=Indexed Equality ("a":2)
FLE2EncryptionPlaceholder_IndexKeyId=12345678123498761234123456789012 ("ki":{"$binary":{"base64":"EjRWeBI0mHYSNBI0VniQEg==","subType":"04"}})
FLE2EncryptionPlaceholder_UserKeyId=12345678123498761234123456789012 ("ku":{"$binary":{"base64":"EjRWeBI0mHYSNBI0VniQEg==","subType":"04"}})
FLE2EncryptionPlaceholder_Value=123456 ("v":123456)
FLE2EncryptionPlaceholder_MaxContentionCounter=0 ("cm":0)
